packages/engine/Source/DataSources/StaticGroundGeometryPerMaterialBatch.js

```diff
@@ -71,6 +61,12 @@ Batch.prototype.isMaterial = function (updater) {
   return defined(material) && material.equals(updaterMaterial);
 };
 
+/**
+ * Adds an updater to the Batch, and signals for a new Primitive to be created on the next update.
+ * @param {JulianDate} time
+ * @param {GeometryUpdater} updater
+ * @param {GeometryInstance} geometryInstance
```

Comment:
When adding documentation to `private` class members (private here meaning anything not part of the public API) make sure to add the `@private` tag. Otherwise, even if the top-level class is already marked as private, it can show up in the generated reference documentation. (Since `Batch` is not exported by this file, I don't think it will show up. But good to be explicit and get in the habit.)

```suggestion
 * @param {GeometryInstance} geometryInstance
 * @private
```